docs/usage.adoc

```diff
@@ -91,11 +91,41 @@ The subscriptions are implemented using Websockets, providing instant and high p
 .Listen to server events for the specific site
 ----
 subscription {
-  event
+  event {
+    type
+    dataAsJson
+  }
+}
+----
+
+Starting from image:images/v-500.svg[Since verision,opts=inline] Guillotine listens node events only for a specific repository and branch of the site.
+
+.Provide `branch`, `repositoryId` and `siteId` to `data` object for WebSocket request:
+
+[source,javascript]
+----
+const portalLib = require('/lib/xp/portal');
+
+webSocket: {
+    data: {
+        branch: req.branch,
+        repositoryId: req.repositoryId,
+        siteId: portalLib.getSite()._id
+    },
+    subProtocols: ['graphql-ws']
+}
+----
+
+or to use the `guillotineLib.createWebSocketData(req)` method which is available starting from image:images/v-5.1.0.svg[Since verision,opts=inline].
+
+[source,javascript]
+----
+webSocket: {
+    data: guillotineLib.createWebSocketData(req),
+    subProtocols: ['graphql-ws']
 }
 ----
 
-Starting from image:images/v-500.svg[Since verision,opts=inline] Guillotine listens events only for a specific `repository` and `branch` which are determined from a site context.
 By default will be listened `node.*` events. In order to configure which events should be listened by application or site you must to specify the `subscriptionEventTypes` option on schema creation, in this case default config will be overwritten.
```

Comment:
This we need to fix as well.

_Only `node.*` events are listened to by default. In order to configure which events should be listened to by an application or a site the `subscriptionEventTypes` option must be specified on schema creation, in this case default config will be overwritten._